cardano-cli/src/Cardano/CLI/EraBased/Governance.hs

```diff
@@ -17,18 +19,17 @@ import           Data.Text (Text)
 import           Options.Applicative hiding (help, str)
 import qualified Options.Applicative as Opt
 
-
 data GovernanceCmd
   = GovernanceVoteCmd VoteCmd
   | GovernanceActionCmd ActionCmd
   | GovernanceMIRPayStakeAddressesCertificate
-      AnyShelleyBasedEra
+      AnyShelleyToBabbageEra
       MIRPot
       [StakeAddress]
       [Lovelace]
       (File () Out)
   | GovernanceMIRTransfer
-      AnyShelleyBasedEra
+      AnyShelleyToBabbageEra
```

Comment:
Why? This is additional churn we're introducing for a command that is going to be removed anyways. Failing on specification of Conway is enough.